### ISSUE TITLE
Wayfire crash with focus-steal-prevent plugin

### ISSUE THREAD

**Issue by marcusbritanicus** [WayfireWM/wayfire-plugins-extra#221]
Basically the title. 

**Wayfire version**: git
**Installation mode**: wf-install

**How to reproduce:**
1. Install wayfire and wayfire-plugins-extra
2. Enable `focus-steal-prevent` plugin (Preferably, enable only this)
3. Run wayfire: it crashed upon loading

**Expected result:**
Wayfire does not crash.

**Stack trace**
1. https://0x0.st/HghB.log - Stack trace without any changes to Wayfire/WPE
2. https://0x0.st/HghM.log - `wf::option_wrapper` and `wf::view_matcher` were moved outside the class as in this [diff](https://0x0.st/Hghc.diff) on @soreau's suggestion.

**Note:** @soreau was not able to repro this. I have not faced this on the previous laptop I owned and the PC I am using. This is issue occurs with the new laptop I bought (Ryzen 7 7730U processor).

**Comment by ammen99** [comment]
I give it a 99% chance that you have a mismatched wayfire and plugin versions, or compiled with different compilers, or something of the sort :)

**Comment by marcusbritanicus** [comment]
@ammen99 Everything was installed it using wf-install. If there is a mismatch, then that's a new bug in wf-install as well... :P 

But no. I have tested it and am 100% sure that there is no plugin version mismatch. It's a fresh install on a fresh system - I have installed it three different ways in three different locations.
1. From git, the usual way as a release build (`/usr/`)
2. From git, the usual way, but debug build (`/tmp`)
3. wf-install to `/opt`

Before each install, I have nuked the previous installs.

Besides, other plugins from wpe work fine, for example workspace-names. If there was any mismatch, it would have affected those as well.
